src/pybind/mgr/dashboard/README.rst

```diff
@@ -69,6 +69,42 @@ The password will be stored as a hash using ``bcrypt``.
 
 The Dashboard's WebUI should then be reachable on TCP port 8080.
 
+Enabling the Object Gateway management frontend
+-----------------------------------------------
+
+If you want to use the Object Gateway management functionality of the
+dashboard, you will need to provide credentials. If you do not have a user
+which shall be used for providing those credentials, you will also need to
+create one::
+
+  $ radosgw-admin user create --uuid=<user> --display-name=<display-name> \
```

Comment:
The correct option is `--uid`, not `--uuid`. Also, I suggest replacing `<user>` with `<id>`, to be consistent with the command line help of the `radosgw-admin` command.